README.md

```diff
@@ -338,7 +337,7 @@ The underlying key types RSA, ECDSA, and EdDSA MUST be supported. Use of ECDSA i
 
 The `nbf` field is OPTIONAL. When omitted, the token MUST be treated as valid beginning from the Unix epoch. Setting the `nbf` field to a time in the future MUST delay using a UCAN. For example, pre-provisioning access to conference materials ahead of time but not allowing access until the day it starts is achievable with judicious use of `nbf`.
 
-The `exp` field MUST be set. If the time is in the past, the token MUST fail validation.
+The `exp` field MUST be set. Following the [principle of least authority](https://en.wikipedia.org/wiki/Principle_of_least_privilege), it is RECOMMENDED to give a timestamp expiry for UCANs. If the token explitely never expires, the `exp` field MUST be set to `null`. If the time is in the past at validation time, the token MUST be treated as expired and invalid.
```

Comment:
Added the option to use an explicit `null` to the expiry. The other option that we discussed was setting it to super far in the future (e.g. `9999999999`), which is roughly what we do today even if it doesn't capture the intention.

We want this field to be fairly salient — no expiry is potentially dangerous and not listing it makes that easy to miss — hence not having it be optional.